components/TARGET_PSA/TESTS/compliance_crypto/test_c024/test_c024.c

```diff
@@ -90,12 +90,18 @@ int32_t psa_aead_encrypt_test(security_t caller)
         TEST_ASSERT_EQUAL(status, PSA_SUCCESS, TEST_CHECKPOINT_NUM(5));
 
         if (is_buffer_empty(check1[i].nonce, check1[i].nonce_length) == TRUE)
+        {
             nonce = NULL;
+            check1[i].nonce_length = 0;
```

Comment:
if you set size as zero,
why leave the .nonce_length) == TRUE ?
it should be false, it doesn't make much sense leaving it as true